src/genegraph/source/graphql/core.clj

```diff
@@ -91,7 +95,7 @@
                       :resolve resource/label
                       :description "Gene symbol"}
               :alternative_label {:type 'String
-                                  :resolve resource/alternative-label
+                                  :resolve gene/alternative-label
```

Comment:
Is there a reason to prefer gene-specific selection of an alternative-label vs making one common to all resources?

Reply:
No, until now I didn't realize the ubiquity of the use of that property.